docs/CHANGELOG.md

```diff
@@ -17,6 +17,14 @@ title: Changelog
 
     *Joel Hawksley*
 
+* Clean up grammar in documentation.
+
+    *Joel Hawksley*
+
+* The ViewComponent team at GitHub is hiring! We're looking for a Rails engineer with accessibility experience: [https://boards.greenhouse.io/github/jobs/3802139]. Reach out to joelhawksley@github.com with any questions!
```

Comment:
@joelhawksley The link seems broken.

Reply:
Looks like this is the active URL: https://boards.greenhouse.io/github/jobs/4020166